src/en/news/blog/2024/v17-2-8-quincy-released/index.md

```diff
@@ -0,0 +1,1452 @@
+---
+title: "v17.2.8 Quincy released"
+date: "2024-11-18"
+author: "Yuri Weinstein"
+tags:
+  - "release"
+  - "quincy"
+---
+
+This is the eighth backport release in the Quincy series. We recommend all users update to this release.
+
+## Notable Changes
+
+- RADOS: `get_pool_is_selfmanaged_snaps_mode` C++ API has been deprecated
+  due to being prone to false negative results.  It's safer replacement is
+  `pool_is_in_selfmanaged_snaps_mode`.
+- RBD: When diffing against the beginning of time (`fromsnapname == NULL`) in
+  fast-diff mode (`whole_object == true` with `fast-diff` image feature enabled
+  and valid), diff-iterate is now guaranteed to execute locally if exclusive
+  lock is available.  This brings a dramatic performance improvement for QEMU
+  live disk synchronization and backup use cases.
+- RBD: The option ``--image-id`` has been added to `rbd children` CLI command,
+  so it can be run for images in the trash.
+- RBD: `RBD_IMAGE_OPTION_CLONE_FORMAT` option has been exposed in Python
```

Comment:
s/`RBD_IMAGE_OPTION_CLONE_FORMAT` option/The `RBD_IMAGE_OPTION_CLONE_FORMAT` option/